snapcraft/internal/project_loader/_extensions/flutter_master.py

```diff
@@ -0,0 +1,131 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2018-2019 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+# Import types and tell flake8 to ignore the "unused" List.
+
+import textwrap
+from typing import Any, Dict, Tuple
+
+from ._extension import Extension
+
+_PLATFORM_SNAP = dict(core18="gnome-3-28-1804")
+
+
+class ExtensionImpl(Extension):
+    """This extension eases creation of snaps that use Flutter
+
+    It tracks the master channel for flutter.
+
+    At build time it ensures the right build dependencies are setup and for
+    the runtime it ensures the application is run in an environment catered
+    for Flutter applications.
+
+    It configures each application with the following plugs:
+
+    \b
```

Comment:
why are there these "\b" lines in the comment?

Reply:
For help to separate the paragraphs when running "snapcraft help"